docs/how-to-use-monty/tutorials/implementing-actions.md

```diff
@@ -0,0 +1,161 @@
+---
+title: Implementing Actions
+---
+
+# What Are Actions?
+
+**Actions** are how Monty moves **Agents** in its **Environment**. Monty's **Motor System** outputs **Actions**, which are then actuated within the **Environment**. An **Agent** is what Monty calls its end effectors.
+
+**Actions** are coupled to what can be actuated in the **Environment**. For example, if a simple robot can only go forward and backward, then the meaningful actions for that robot are restricted to going forward and backward. The robot environment would be unable to actuate a `Jump` action.
+
+With that in mind, before creating a new **Action**, consider what can be actuated in the **Environment**.
+
+# Action Lifecycle
+
+At a high level, **Actions** are created by the **Motor System** and actuated by the **Environment**.
+
+![](../../figures/how-to-use-monty/tutorials/action_lifecycle_high_level.png)
+
+Within the **Motor System**, **Policies** either explicitly choose a specific **Action** by creating it directly, or sample from a pool of actions available.
```

Comment:
Is "sample" similar to clone?  Is that the reason you would do it?  Just to not have to create a new action?